scipy/special/boost_special_functions.h

```diff
@@ -1495,3 +1495,118 @@ hypergeom_skewness_double(double n, double N, double M)
 }
 
 #endif
+
+template<typename Real>
+Real
+landau_pdf_wrap(const Real x, const Real loc, const Real scale)
+{
+    if (std::isfinite(x)) {
```

Comment:
@dschmitz89 I just copied from your `_nct_pdf` PR. I'm not sure what this function does at infinity, so it might be fine to eliminate this, right?

Reply:
To be honest, I think I just also copied it from the non-central f distribution. As long as there are no related issues, I would opt for uniformity.